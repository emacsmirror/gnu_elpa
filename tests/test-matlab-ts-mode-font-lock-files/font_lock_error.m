% -*- matlab-ts -*-
function font_lock_error(foo)
    out = ['''(', newline];

    for x = 1:length(foo)
        c = foo(x);
        if regexp(c, '\(variable|slxFile\)"\)', 'once')
        else
            m = regexp(c, '^[ \t]*\(("[^"]+")[ \t]+\.[ \t]+".*\(mFile|pFile|mex\)"\)', tokens)
            if ~isempty(m)
                out = [out, ...
            end
        end
    end
end


