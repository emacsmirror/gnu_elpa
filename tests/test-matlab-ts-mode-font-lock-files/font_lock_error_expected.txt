c ccc ccccccccc ccc
kkkkkkkk fffffffffffffffbvvvb
    vvv o bSsssSD BBBBBBBbD

    kkk v o nDBBBBBBbdddb
        v o dddbdbD
        kk BBBBBBbdD SsssssssssssssssssssssssSD SssssSb
        kkkk
            v o BBBBBBbdD Sss EEssssssssssssss EEsssss EEsssssssssssssssssssssssssssSD ddddddb
            kk oBBBBBBBbdb
                www w wwwww
            kkk
        kkk
    kkk
kkk


